packages/now-cli/src/index.js

```diff
@@ -135,7 +138,15 @@ const main = async argv_ => {
     console.log(
       info(
         `${chalk.bgRed('UPDATE AVAILABLE')} ` +
-        `Run ${cmd(await getUpdateCommand())} to install Now CLI ${update.latest}`
+          `Run ${cmd(await getUpdateCommand())} to install Now CLI ${
+            update.latest
+          }`
+      )
+    );
+
+    console.log(
+      info(
+        `Changelog: https://github.com/zeit/now/releases/tag/now@${update.latest}`
```

Comment:
@styfle We should make sure to always use the `now` tag for the release when it includes a Now CLI update.

Reply:
I wonder if there's anything we can do to more reliably enforce that. Seems prone to user error.

Reply:
Well, at least it won't 404 (if it's a tag, and not a release), since the URL works all the same: https://github.com/zeit/now/releases/tag/now@16.1.4-canary.5

Though still, an empty changelog isn't very user-friendly.